app/views/layouts/application.html.erb

```diff
@@ -28,7 +28,9 @@
 
 <% content_for :content do %>
   <%= render 'layouts/staging_ribbon' if Rails.env.staging? %>
-  <%= yield %>
+  <main role="main" tabindex="-1" id="<%= 'content' unless current_page? root_path %>">
```

Comment:
What's special about the root path? Should the application layout know about rendering behaviour at a given path?

Reply:
I don't know. I lazily copied the implementation over from Publish. Shame on me. :)